source_lists/wesnoth

```diff
@@ -247,9 +247,6 @@ gui/dialogs/unit_recruit.cpp
 gui/dialogs/wml_error.cpp
 gui/dialogs/wml_message.cpp
 gui/dialogs/surrender_quit.cpp
-gui/widgets/addon_list.cpp
-gui/widgets/button.cpp
-gui/widgets/chatbox.cpp
```

Comment:
Looks like you need to rebase to current master.

Reply:
That's the result of merging master. I just re-checked and those files are not in the source list of master